### PR TITLE
feat(wallet): centered text blocks

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves
Implements the centered text blocks

## Figma
https://www.figma.com/design/qtXXlwdslAze6yNbLLzIRa/Website-redesign-2.0?node-id=15367-15363&node-type=frame&t=11ZCWH0Op36rSecb-0

Screenshot 2024-11-06 at 14 59 56

Screenshot 2024-11-06 at 14 59 40

### CODE REVIEW

File: src/content/wallet.json [hunk 1]
```diff
@@ -247,15 +247,48 @@
       }
     ]
   },
+  {
+    "title": "<b>Get the Latest</b> on Safe{Wallet}",
+    "text": "Sign up to our updates and be informed about new features, improvements, and special announcements.",
+    "buttons": [
+      {
+        "text": "Subscribe",
+        "href": "https://safe.global",
```
Comment (DiogoSoaress) [inline src/content/wallet.json:256]: Waiting for Danilo to provide the real link

File: src/content/wallet.json [hunk 2]
```diff
@@ -247,15 +247,48 @@
...
+    "text": "Still have questions and want to know more about Safe {Wallet}? Watch our demo.",
+    "buttons": [
+      {
+        "text": "Launch wallet",
+        "href": "https://app.safe.global",
+        "variant": "button"
+      },
+      {
+        "text": "View demo",
+        "href": "https://safe.global",
```
Comment (DiogoSoaress) [inline src/content/wallet.json:276]: Waiting for the real URL

File: src/components/Wallet/ImageGradient/index.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { Container, Divider } from '@mui/material'
+import type { BaseBlock } from '@/components/Home/types'
+import css from './styles.module.css'
+
+const ImageGradient = ({ image }: Partial<BaseBlock>) => (
+  <Container className={css.container}>
+    <img src={image?.src} alt={image?.alt} />
```
Comment (iamacook) [inline src/components/Wallet/ImageGradient/index.tsx:7]: If there's no image, should this be rendered? Might make sense to return `null` instead.

File: src/content/wallet.json [hunk 4]
```diff
@@ -247,15 +247,48 @@
...
+        "variant": "button"
+      }
+    ],
+    "component": "common/CenteredTextBlock"
+  },
   {
     "component": "commonCMS/Faq"
   },
+  {
+    "title": "Create your <b>smart account</b> now",
```
Comment (iamacook) [inline src/content/wallet.json:266]:
```suggestion
    "title": "Create your <b>Smart Account</b> now",
```

File: src/content/wallet.json [hunk 5]
```diff
@@ -247,15 +247,48 @@
...
+    "text": "Still have questions and want to know more about Safe {Wallet}? Watch our demo.",
+    "buttons": [
+      {
+        "text": "Launch wallet",
```
Comment (iamacook) [inline src/content/wallet.json:270]:
```suggestion
        "text": "Launch Safe{Wallet}",
```

File: src/components/Wallet/ImageGradient/index.tsx (hunk 3 above)
Comment (DiogoSoaress) [reply src/components/Wallet/ImageGradient/index.tsx]: I added an early return if the component doesn't receive `image`

**Review by iamacook — APPROVED** [review]

**Pull request merged**